website/docs/d/fargate_workload_agent.md

```diff
@@ -32,7 +31,6 @@ data "sysdig_fargate_workload_agent" "instrumented_containers" {
 ## Argument Reference
 
 * `container_definitions` - (Required) The input Fargate container definitions to instrument with the Sysdig workload agent.
-* `sysdig_access_key` - (Required) The Sysdig Access Key (Agent token).
 * `orchestrator_host` - (Required) The orchestrator host to connect to.
 * `orchestrator_port` - (Required) The orchestrator port to connect to.
```

Comment:
Just a heads up

These arguments are marked as required, but currently, that's not really enforced.